config/reference_container.yaml

```diff
@@ -0,0 +1,8 @@
+ 
+reference_container_folder: "/home/sven/Documents/reference_container/"
```

Comment:
this is a no-go: a path to a developer-local folder will never work on somebody else's computer, and thus cannot be part of the repository

Reply:
Sorry, that was accidentally still in the folder (for testing purposes) when I made the commit.